TbsCore/Helpers/GithubHelper.cs

```diff
@@ -1,6 +1,6 @@
-﻿using System;
+﻿using Octokit;
```

Comment:
What's this?

Reply:
ctrl + k/ctrl + e cause this, just sort using